code/script.js

```diff
@@ -1,47 +1,152 @@
 // All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById('input-wrapper')
+const nameInput = document.getElementById('name-input')
+const nameForm = document.getElementById('name-form')
+// Button-Sounds
+const voff = document.getElementById('voff')
+const voffSound = document.getElementById('voff-sound')
+const mjauAlert = document.getElementById('mjau')
 
-// Global variables, if you need any, declared here
+let currentQuestion = 1
 
-// Functions declared here
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   if (sender === 'user') {
+    console.log("I am user!")
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/user_2.png" alt="User" />  
       </section>
     `
   } else if (sender === 'bot') {
+    console.log("I am bot!")
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/bot_2.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+// This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Questions
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  currentQuestion = 1
+    showMessage(`Hello there, What's your name?`, 'bot')
+}
+const dogQuestion = () => {
+  currentQuestion++
+  setTimeout(() => showMessage('Do you like dogs', 'bot'), 1000)
+    
+    inputWrapper.innerHTML =  `
+    <button id="yes">YES</button>
+    <button id="no">NO</button>
+    `
+ document.getElementById('yes').addEventListener('click', () => {
+    showMessage('Yes', 'user')
+    setTimeout(() => showMessage('That´s what I thought! I mean who doesn’t?!', 'bot'), 500)
+    handleInput()
+  })
+  document.getElementById('no').addEventListener('click', () => {
+    showMessage('No', 'user')
+    setTimeout(() => showMessage('Wrong answer! Who are you?! A cat person?', 'bot'), 1000)
+    setTimeout(() => showMessage('Try another Bot! Bye bye! 👋🏼', 'bot'), 1500)
```

Comment:
Fun conversation! We could try too deelay some messages a bit more for more pleasant reading.